src/Model/PowerElectronics/InductionMotor/InductionMotor.cpp

```diff
@@ -17,6 +17,12 @@ namespace GridKit {
  * Calls default ModelEvaluatorImpl constructor.
  * @todo create a test case utilizing the component.
  * @todo create a unit test to check correctness of component
+ * 
+ * @tparam ScalarT - floating point type for the model
+ * @tparam IdxT - integer index type for the model
```

Comment:
It does not need to be a floating point type.
```suggestion
 * @tparam ScalarT - data type for scalar variables in the model
 * @tparam IdxT - integer index type for the model
```